### PR TITLE
Fixes #3268 - Refactor image uploading in the new form

### LINKED ISSUE #3268: Add image uploading functionality

### PR DESCRIPTION
**Pull request by ksy36**
r? @miketaylr

### CODE REVIEW

File: webcompat/static/js/lib/wizard/steps/upload-helper/upload-text.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import notify from "../../notify.js";
+
+const changeUploadText = textId => {
```
Comment (miketaylr) [inline webcompat/static/js/lib/wizard/steps/upload-helper/upload-text.js:7]: I guess the newest prettier is going to complain about this right? Maybe good to rebase against master and re-run prettier?

File: webcompat/static/js/lib/wizard/tests/validation.test.js [hunk 2]
```diff
@@ -1,4 +1,9 @@
-import { isUrlValid, isEmpty } from "../validation.js";
+import {
+  isUrlValid,
+  isEmpty,
+  isImageTypeValid,
+  blobOrFileTypeValid
```
Comment (miketaylr) [inline webcompat/static/js/lib/wizard/tests/validation.test.js:5]: (prettier probably wants an extra comma here too, etc)

File: webcompat/static/js/lib/wizard/utils.js [hunk 3]
```diff
@@ -18,3 +18,48 @@ export const charsPercent = (value, min = 30) => {
...
+
+  img.src = dataURI;
+};
+
+/*
+  If the users browser understands the FileReader API, show a preview
+  of the image they're about to load, then invoke the passed in callback
+  with the result of reading the blobOrFile as a dataURI.
+*/
+export const convertToDataURI = (blobOrFile, callback) => {
```
Comment (miketaylr) [inline webcompat/static/js/lib/wizard/utils.js:54]: I guess we can convert these all to async functions, or Promises once we have Babel in place (not worth doing now!).

**Comment by ksy36** [comment]
Thanks! I'll rebase the feature branch to get prettier updates

File: webcompat/static/js/lib/wizard/utils.js (hunk 3 above)
Comment (ksy36) [reply webcompat/static/js/lib/wizard/utils.js:54]: Yeah, good point :)

**Pull request merged**